### DISCUSSION TITLE
How to actually use dockerized patcher?

### DISCUSSION THREAD

**Discussion by cakir-enes** [ryanoasis/nerd-fonts#1020] [Q&A]
Simply, this doesnt output anything meaningful. Tried absolute path, single font instead of folder just outputs options that I give
```
$ docker run --rm -v $(pwd)/in:/in -v $(pwd)/out:/out nerdfonts/patcher --complete
Running with options:
 --complete
 ```
 
 I just want to patch this font https://experi.mentallydesigned.com/type/bam/ without installing stuff.

**Comment by Finii** [comment]
This is how I would do it (note that I have `docker` only halfhearted set up so that I need `sudo`):

![image](https://user-images.githubusercontent.com/16012374/205785102-d06c9121-c8ee-405b-8a5d-ddfc7d0d53be.png)

[...]

![image](https://user-images.githubusercontent.com/16012374/205785219-4f35b81c-0f42-4d70-8bfa-4cd2df83c0af.png)

I do not really see what you did wrong, though. Is there no more detailed error message or something.
